test/system/helpers.bash

```diff
@@ -341,6 +341,16 @@ function is_cgroupsv2() {
     test "$cgroup_type" = "cgroup2fs"
 }
 
+# True if podman is using netavark
+function is_netavark() {
+    local be_type
+    be_type=$($PODMAN info --format '{{.Host.NetworkBackend}}')
+    if [[ "$be_type" =~ netavark ]]; then
+        return 0
+    fi
+    return 1
```

Comment:
Could be shortened and legibilitized as just
```
test "$be_type" =~ "netavark"
```

Reply:
I'll change it so it's consistent with the style of other functions.  However, my understanding is that `[[` is a bash built-in,  therefor likely MUCH faster than executing the `test` binary.  I'd also argue that keeping things less readable in this instance is actually an asset. Since it keeps the bash-charlatans and bash-wannabe's far away :rofl: Lol, bash, legible, funny stuff :rofl: